packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -166,10 +166,6 @@ describe('getBinaryTargetForCurrentPlatformInternal', () => {
         }),
       ).toBe('debian-openssl-3.0.x')
       expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
-      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
-        prisma:warn Prisma doesn't know which engines to download for the Linux distro "unknown". Falling back to Prisma engines built "debian".
-        Please report your experience by creating an issue at https://github.com/prisma/prisma/issues so we can add your distro to the list of known supported distros.
-      `)
```

Comment:
I am ok removing this if in case of failure (engine loading) we do display the necessary information for the user to understand we defaulted to debian because xyz

Reply:
So, should this be moved to the logic that tests for panics?

Reply:
Not sure if it is a panic, but I think we have some code to try/catch `dlopen` maybe it should happen there?